pkg/operation/botanist/component/backupentry/backupentry.go

```diff
@@ -163,6 +135,77 @@ func (b *backupEntry) Wait(ctx context.Context) error {
 	})
 }
 
+// Migrate uses the garden client to deschedule the BackupEntry from its current seed.
+func (b *backupEntry) Migrate(ctx context.Context) error {
+	backupEntry := &gardencorev1beta1.BackupEntry{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      b.values.Name,
+			Namespace: b.values.Namespace,
+		},
+	}
+
+	if err := b.client.Get(ctx, kutil.Key(b.values.Namespace, b.values.Name), backupEntry); err != nil {
+		return client.IgnoreNotFound(err)
+	}
+
+	metav1.SetMetaDataAnnotation(&backupEntry.ObjectMeta, v1beta1constants.GardenerOperation, v1beta1constants.GardenerOperationMigrate)
+	metav1.SetMetaDataAnnotation(&backupEntry.ObjectMeta, v1beta1constants.GardenerTimestamp, TimeNow().UTC().String())
+	backupEntry.Spec.SeedName = b.values.SeedName
```

Comment:
Wouldn't changing the seed name be good enough?

Reply:
Added this for consistency reasons as we add the `Restore` and `Reconcile` operation annotations in the respective functions.

Reply:
Hm, I can't follow - why is this an argument for putting some annotations while there's already a better way to trigger the operation (changing the seed name)? We don't do the same for `Shoot`s either, do we?